### PR TITLE
docs: correct spelling in docs

### PR DESCRIPTION
**Pull request by vbelouso**
Correct spelling to improve the readability of the documentation

### CODE REVIEW

File: docs/design/openstack/networking-infrastructure.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Keepalived. While the bootstrap node is up, it will have priority running the AP
...
...ially, the bootstrap node has the highest priority for hosting the API VIP, so they will point to addresses there at startup. Meanwhile, the master nodes will try to get the control plane, and the OpenShift API up. Keepalived implements periodic health checks for each VIP that are used to determine the weight assigned to each server. The server with the highest weight is assigned the VIP. Keepalived has two separate healthchecks that attempt to reach the OpenShift API and CoreDNS on the localhost of each master node. When the API on a master node is reachable, Keepalived substantially increases it's weight for that VIP, making its priority higher than that of the bootstrap node and any node that does not yet have the that service running. This ensures that nodes that are incapable of serving DNS records or the OpenShift API do not get assigned the respective VIP. The Ingress VIP is also managed by a healthcheck that queries for an OCP Router HAProxy healthcheck, not the HAProxy we stand up in  static pods for the API. This makes sure that the Ingress VIP is pointing to a server that is running the necessary OpenShift Ingress Operator resources to enable external access to the node.
```
Comment (e-tienne) [inline docs/design/openstack/networking-infrastructure.md:51]: What's the spelling issue here?

**Review by e-tienne — COMMENTED** [review]
The spelling corrections look good to me; however I'd like to understand the change in `docs/design/openstack/networking-infrastructure.md`

File: docs/design/openstack/networking-infrastructure.md (hunk 1 above)
Comment (vbelouso) [reply docs/design/openstack/networking-infrastructure.md:51]: it words `seperate`. I'm not sure why the whole block of text is listed in `diff` and not a specific word.

**Review by e-tienne — COMMENTED** [review]
/lgtm

**Review by staebler — APPROVED** [review]
/lgtm

**Pull request merged**